src-tauri/src/main.rs

```diff
@@ -345,3 +346,16 @@ async fn get_server_player_count() -> Result<(i32, usize), String> {
 
     Ok((total_player_count, server_count))
 }
+
+#[tauri::command]
+/// Spawns repair window
+async fn open_repair_window(handle: tauri::AppHandle) -> Result<(), String> {
+    let _repair_window = tauri::WindowBuilder::new(
```

Comment:
You should name the window something different than "tao window".

Reply:
Done in ce8717f314e8c4f85169f712e75d99b97845dc65

It's now called `FlightCore Repair Window`